utils/miscellaneous.go

```diff
@@ -48,3 +49,22 @@ func MergeStringMap(map1 Set, map2 Set) Set {
 
 	return out
 }
+
+// DataFromSliceOrFile returns data from the slice (if non-empty), or from the file,
+// or an error if an error occurred reading the file
+func DataFromSliceOrFile(data []byte, file string) ([]byte, error) {
+	if len(data) > 0 {
+		return data, nil
+	}
+
+	if len(file) > 0 {
+		fileData, err := ioutil.ReadFile(file)
```

Comment:
Since go 1.16 `ioutil` in most cases simply calls the corresponding`os` function. I think we should use `os.ReadFile`.

Reply:
✅